### PR TITLE
Fix bugs found when parsing RTS-GMLC

### PR DESCRIPTION
**Pull request by daniel-thom**

### CODE REVIEW

**Review by claytonpbarrows — APPROVED** [review]
Thanks!

**Comment by daniel-thom** [comment]
@claytonpbarrows Unit tests on the master branch failed locally for the same reasons as CI. Do you know what's going on?

**Comment by claytonpbarrows** [comment]
This happened: https://github.com/lanl-ansi/PowerModels.jl/issues/798

File: test/test_wrapper.jl [hunk 1]
```diff
@@ -6,6 +6,17 @@ function bus_gen_values(data, solution, value_key)
     return bus_pg
 end
 
+function parse_pm_pmi(file)
+    pm_data = PowerModels.parse_file(file)
+    pmi_data = PMI.get_pm_data(System(file))
+    #TODO: remove the following for loop once PM#v0.20.0 is tagged
+    for id in keys(pmi_data["bus"])
+        pm_data["bus"][id] = pmi_data["bus"][id]
+    end
```
Comment (claytonpbarrows) [inline test/test_wrapper.jl]: @daniel-thom this is the short term hack that fixes the parsing discrepancy between PM and PSY that is planned to be addressed in PM#0.20.0

**Pull request merged**